sci-mathematics/mathematica/mathematica-13.0.1.ebuild

```diff
@@ -0,0 +1,140 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHECKREQS_DISK_BUILD=19G
+inherit desktop unpacker xdg
+
+DESCRIPTION="Wolfram Mathematica"
+SRC_URI="Mathematica_${PV}_BNDL_LINUX.sh"
+HOMEPAGE="https://www.wolfram.com/mathematica/"
+
+LICENSE="all-rights-reserved"
+KEYWORDS="-* ~amd64"
+SLOT="0"
+IUSE="cuda +doc"
+
+RESTRICT="strip mirror bindist fetch"
+
+# Mathematica comes with a lot of bundled stuff. We should place here only what we
+# explicitly override with LD_PRELOAD.
+# RLink (libjri.so) requires dev-lang/R
+RDEPEND="
+	dev-lang/R
```

Comment:
Do we need R for the whole thing? I think only some of the installed binaries link to R, so would it be possible to avoid pulling in R if we do not want to use those files?

Reply:
This update splits a R use flag, which will install the R language when the user needs to use RLink.